src/test/resources/conrefbreaksxref/exp/preprocess/conrefbreaksxref.dita

```diff
@@ -3,20 +3,20 @@
 <shortdesc class="- topic/shortdesc ">This topic conrefs to another, which uses xref internally.</shortdesc>
 <body class="- topic/body ">
 <section class="- topic/section "><title class="- topic/title ">Section that is pulled to another topic.</title><p class="- topic/p ">Look
-as the figure in this section: <xref class="- topic/xref " href="#conrefbreaksxref/d7e48" type="fig"><?ditaot gentext?>Figure 1</xref>.</p><p class="- topic/p ">Look
-at a figure in this topic, outside of this section: <xref class="- topic/xref " href="#conrefbreaksxref/d7e56"/></p><p class="- topic/p ">Now
+as the figure in this section: <xref class="- topic/xref " href="#conrefbreaksxref/d8e48" type="fig"><?ditaot gentext?>Figure 1</xref>.</p><p class="- topic/p ">Look
+at a figure in this topic, outside of this section: <xref class="- topic/xref " href="conrefbreaksxreftarget.dita#this-target/out" type="fig"><?ditaot gentext?>Figure 2</xref></p><p class="- topic/p ">Now
```

Comment:
This is an example of a case where the old output was broken local link and the new output is link to the original conref target. That may or may not actually be live based on whether the conref target is also output. So in most cases, the link will be broken in e.g. HTML5 output but in a different way than it used to be.